config/initializers/carrier_wave.rb

```diff
@@ -0,0 +1,15 @@
+CarrierWave.configure do |config|
+  if Rails.env.staging? || Rails.env.production?
+    config.storage = :fog
+    config.fog_provider = 'fog/aws'                        # required
+    config.fog_credentials = {
+      provider:              'AWS',                        # required
+      aws_access_key_id:     'xxx',                        # required
```

Comment:
These settings should be configured. *Make sure* you do it using environment variables, **do not** put your access info in your repo.